rust/perspective-viewer/src/rust/custom_elements/viewer.rs

```diff
@@ -617,4 +617,21 @@ impl PerspectiveViewerElement {
     pub fn unsafe_get_model(&self) -> *const PerspectiveViewerElement {
         std::ptr::addr_of!(*self)
     }
+
+    /// Opens the column edit sidebar for a specific column.
+    /// The Viewer component will then send a
+    /// "perspective-toggle-column-settings" event to the plugin with the
+    /// currently selected column. The plugin should respond to the event to
+    /// highlight the selected column appropriately.
```

Comment:
These are the API docs which are published to https://perspective.finos.org/docs/obj/perspective-viewer/ , and unlike the rest of the repo, they should be written from the point-of-view of a developer using `<perspective-viewer>` in their application, not a developer working on the Perspective codebase itself.

Reply:
I updated the docs to something I think is more user-friendly. Let me know if it needs more changes.